labelbox/client.py

```diff
@@ -208,9 +213,9 @@ def check_errors(keywords, *path):
         if internal_server_error is not None:
             message = internal_server_error.get("message")
 
-            if message.startswith("Syntax Error"):
+            if message.startswith("Syntax Error") or message.startswith(
+                    "Invite(s) cannot be sent"):
```

Comment:
This is because the backend just throws a 500 and there is no way to handle this without parsing the message (see the comment just above this.

Server side validation is one reason we raise this exception:
https://github.com/Labelbox/labelbox-python/blob/56a366ce102b216a278993908863817282103955/labelbox/exceptions.py#L63-L65

I don't like this pattern either but I don't think fixing this is within the scope of this delivery.

Reply:
aite, one small nit is that `startswith` takes a tuple I believe
`message.startsiwh(("message1, "message"))`

Reply:
Fixed.